srcpkgs/RustScan/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'RustScan'
+pkgname=RustScan
+version=2.0.1
+revision=1
+build_style=cargo
+hostmakedepends="python3 perl"
```

Comment:
What is it building internally that requires python and perl? Could you be missing `openssl-devel` in `makedepends`?

Reply:
Nothing, you are right. Fixed.